README.md

```diff
@@ -45,3 +45,27 @@ To increase the version of all modules and pom files, run the following commands
 mvn versions:set -DnewVersion="0.1.0-preview02"
 mvn versions:commit
 ```
+
+### Debug Java application or Dapr's Java SDK
+If you have a Java application or an issue on this SDK that needs to be debugged, follow the steps below:
+
+Install [Pen Load Balancer](https://sourceforge.net/projects/penloadbalancer/) or your preferred load balancer utility:
+```sh
+sudo apt-get install pen
+```
+Note: Pen is also available on Windows and MacOS.
```

Comment:
I pen is available as a chocolatey or brew install , consider adding them as well here.
Also please link to https://github.com/dapr/docs/tree/master/howto/vscode-debugging-daprd as another option when using VSCode.

Reply:
For Windows, it is a download. For MacOS you will need to build from source.